src/chapters/modules/the-main-function.rst

```diff
@@ -191,6 +191,8 @@ bugs from loose statements:
    change the window appearance, and call drawing functions.
 #. Using ``main()`` helps prevent shadowing of variables.
 
+.. TODO: Have they encountered variable shadowing before?
```

Comment:
Yes, section 10.8 deals with function scope and variable shadowing.

Reply:
Sounds good! I will remove the comment!